android/src/com/frostwire/android/gui/activities/VPNStatusDetailActivity.java

```diff
@@ -58,35 +60,78 @@ protected void initComponents(Bundle savedInstanceState) {
         
         final ImageView headerIcon = findView(R.id.view_vpn_status_header_icon);
         final TextView headerTitle = findView(R.id.view_vpn_status_header_title);
-        final Button button = findView(R.id.view_vpn_status_get_vpn_button);
+        final TextView unProtectedTitle = findView(R.id.view_vpn_status_header_un_protected);
```

Comment:
sorry will rename. Was using the TextView for displaying both the protected and unprotected text based on your connection. But will change it for better.